prioritizer/prioritizer.py

```diff
@@ -158,76 +159,148 @@ def quantitative_score(city, action):
             "medium": 10,
             "big": 5
         }
-        print(cost_investment_needed)
+        cost_score_weight = weights.get("CostInvestmentNeeded", 1)
         cost_score = cost_score_map.get(cost_investment_needed, 0)
-        score += (cost_score / 15) * SCORE_MAX
-        #ratio = action["CostInvestmentNeeded"] / MAX_COST if action["CostInvestmentNeeded"] else 1
-        #score += (1 - ratio) * SCORE_MAX
+        score += (cost_score / 15) * SCORE_MAX * cost_score_weight
 
+    print("Score after cost:", score)
+    print("-------------")
     return score
 
+class Action(BaseModel):
```

Comment:
These things I would always move to a separate file inside a /utils folder.
Its not really concerned with the actual prioritization but just the LLM call which can be a function to import and makes the file much leaner